core/shared/src/main/scala/zio/logging/LogFormat.scala

```diff
@@ -841,9 +842,11 @@ object LogFormat {
 
   val enclosingClass: LogFormat =
     LogFormat.make { (builder, trace, _, _, _, _, _, _, _) =>
-      trace match {
-        case Trace(_, file, _) => builder.appendText(file)
-        case _                 => builder.appendText("not-available")
+      val parsed = Tracer.instance.parseOrNull(trace)
```

Comment:
You can also use `Trace.parseOrNull`
```suggestion
      val parsed = Trace.parseOrNull(trace)
```

That's the "public" function used in the ZIO codebase. We don't usually use `Tracer.instance.xxx` directly